youtubesearchpython/core/channelsearch.py

```diff
@@ -85,76 +95,3 @@ def result(self, mode: int = ResultMode.dict) -> Union[str, dict]:
         elif mode == ResultMode.dict:
             return {'result': self.response}
 
-
-class ChannelSearchCore(ChannelRequestCore):
-    def __init__(self, query: str, language: str, region: str, searchPreferences: str, browseId: str, timeout: int):
-        super().__init__(query, language, region, searchPreferences, browseId, timeout)
-
-    def _parseChannelSearchSource(self) -> None:
-        try:
-            last_tab = self.response["contents"]["twoColumnBrowseResultsRenderer"]["tabs"][-1]
-            if 'expandableTabRenderer' in last_tab:
-                self.response = last_tab["expandableTabRenderer"]["content"]["sectionListRenderer"]["contents"]
-            else:
-                tab_renderer = last_tab["tabRenderer"]
-                if 'content' in tab_renderer:
-                    self.response = tab_renderer["content"]["sectionListRenderer"]["contents"]
-                else:
-                    self.response = []
-        except:
-            raise Exception('ERROR: Could not parse YouTube response.')
-
-
-class ChannelPlaylistSearchCore(ChannelRequestCore):
```

Comment:
This class, I might have accidentally removed. This class was intended for searching playlists within channel (eg. with a query), right? I can't seem to find this feature anywhere in YouTube webapp. Can you maybe screenshot where exactly can you pick between playlists and videos, as I can only search by everything. If you provide me this, I'll be able to implement this class (or you may for that matter).

Reply:
This was probably just an misunderstanding on my side.